docs/reference/commandline/service_create.md

```diff
@@ -177,6 +177,45 @@ $ docker service create \
   redis:3.0.6
 ```
 
+### Container networking
+The containers provisioned in docker swarm mode can be accessed in service discovery either via a Virtual IP (VIP) and routed through the docker swarm ingress overlay network. Or via a DNS round robbin (DNSRR). This configuration is set using the `--endpoint-mode [vip|dnsrr]` setting. Note that dnsrr is incompatible with `--published` ports.
+
+### Service discovery
+
+Discovering other service from within a container can be ac hived via DNS lookup, but you will get different results depending on whether you want the actual service IPs, or the VIP's.
+
+```
+~> nslookup service
+
+Address: 10.255.0.7           # this is the VIP on the ingress network
+
+~> nslookup tasks.service
+
+Non-authoritative answer:
+Name:	tasks.service
+Address: 10.255.0.26          # this is the real IP of a service on the ingress network
```

Comment:
I don't think it is correct to state that this is the `real` IP of the service. It would be the IP of the container that is a task for the service.